### ISSUE TITLE
SDMXStructure.xsd element "ValueList" has an uppercase L which is inconsistent with "Codelist"

### ISSUE THREAD

**Issue by glennmdt** [glennmdt/sdmx-ml#1]
Priority: low

<xs:element name="ValueList" type="ValueListType">
vs
<xs:element name="Codelist" type="CodelistType">

It would be cleaner if "Valuelist" was used to make it consistent with "Codelist"

**Comment by jgager** [comment]
@glennmdt - I took the camel case ("ValueList") from the model document (line 1943 in version DRAFT-0_7-for-review, for example). I will point out that in the table 10.1.2.2, it is listed as "Valuelist", but I took that to be a mistake. I would also add that while "codelist" as a single term is something I have seen often (especially in this context of a controlled vocabulary), I have never seen "valuelist" as a single word. I am happy to change, but would like to first confirm that the model matches and the rationale behind the change is inline with the actual terminology.
